lua/weapons/arccw_base/cl_viewmodel.lua

```diff
@@ -327,7 +332,7 @@ function SWEP:GetViewModelPosition(pos, ang)
         target.pos = target.pos + coolswaypos
     end
 
-    speed = 1 / self:GetSightTime() * speed * FT
+    speed = 1 / self:GetSightTime() * speed * FT * .5
```

Comment:
this better syncs weapon movement with the lock times dictated by SightTime
tested with ArcCW CS+ weapons as well, no negative effects